CMakeLists.txt

```diff
@@ -346,6 +346,9 @@ if( NOT ${CMAKE_USE_PTHREADS_INIT} )
     message( FATAL_ERROR "Only pthreads supported - thread library found is [${THREADS_LIBRARIES}]" )
 endif()
 
+### vendored dependencies
+add_subdirectory(third-party)
```

Comment:
I've seen these directories typically named "extern" instead (just an observation). In my opinion we should be using git submodules instead, for this purpose, but maybe that's a bridge too far.

Reply:
I don't disagree :)